src/calculateRentalCost.js

```diff
@@ -28,7 +28,18 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const dayRent = 40;
+  const bigDiscount = 50;
+  const smallDiscount = 20;
+  let total = dayRent * days;
```

Comment:
better to rename the variable. And if you'll return values as I've shown you below - you don't have to declare it with let, const will do perfect

```suggestion
  const fullPrice = dayRent * days;
```